themes/src/main/resources-community/theme/base/email/messages/messages_sk.properties

```
@@ -1,6 +1,11 @@
 emailVerificationSubject=Overenie e-mailu
 emailVerificationBody=Niekto vytvoril účet {2} s touto e-mailovou adresou. Ak ste to vy, kliknite na nižšie uvedený odkaz a overte svoju e-mailovú adresu \n\n{0}\n\nTento odkaz uplynie do {1} minút.\n\nAk ste tento účet nevytvorili, ignorujte túto správu.
 emailVerificationBodyHtml=<p>Niekto vytvoril účet {2} s touto e-mailovou adresou. Ak ste to vy, kliknite na nižšie uvedený odkaz na overenie svojej e-mailovej adresy.</p><p><a href="{0}"> Odkaz na overenie e-mailovej adresy </a></p><p>Platnosť odkazu vyprší za {1} minút.</p><p> Ak ste tento účet nevytvorili, ignorujte túto správu.</p>
+orgInviteSubject=Pozvánka do organizácie {0}
+orgInviteBody=Niekto vás pozval do organizácie "{3}". Pre pridanie sa kliknite na odkaz nižšie.\n\n{0}\n\nTento odkaz vyprší za {4}.\n\nAk sa do organizácie nechcete pridať, ignorujte túto správu.
```

```suggestion
orgInviteBody=Niekto Vás pozval do organizácie "{3}". Pre pridanie sa, kliknite na odkaz nižšie.\n\n{0}\n\nTento odkaz vyprší za {4}.\n\nAk sa do organizácie nechcete pridať, ignorujte túto správu.
```
I feel, there should be a comma in all these cases "Pre pridanie sa, kliknite na odkaz nižšie". @mposolda isn't it the same for czech language?
Also, the following occurrences of "vás" should be with capital "V".